Support multiple logos for login pages

kmshin1397 opened the pull request:
![Screenshot 2020-10-09 195957](https://user-images.githubusercontent.com/17696889/95644309-9fdffd80-0a6a-11eb-9824-f35eec8403a9.png)

- Template loginerror.html as well
- Ended up making a separate "login_logos" section in the configs as these can be different from the header bar (due to differences in background color, may need different color versions of the logos)
- Update config.default.yaml accordingly

Note: GROWTH Marshal logo is just an example, was not committed into default configs

stefanv commented:
@kmshin1397 Perhaps we should simply render them on the same color banner instead of complicating the configuration?

kmshin1397 commented:
@stefanv You're right I like that better. I'll work on that

kmshin1397 commented:
@stefanv Done:

![Screenshot 2020-10-10 123155](https://user-images.githubusercontent.com/17696889/95663625-9cde1f00-0af5-11eb-9620-b7c0795ba368.png)

acrellin reviewed (approved):
Looks great. It looks like login.html and loginerror.html still need to be removed, though

kmshin1397 commented on static/loginerror.html.template:
```diff
@@ -0,0 +1,43 @@
+    <div class="loginBox">
+      <div class="loginHeader">
+        <div class="logos">
+            {%- for logo in app.logos %}
```
@stefanv I explored this, but I think it would require changes to baselayer. We'd need to change how we initialize jinja2 template objects to specify a FileLoader with access to the base login page template file in order to extend it in the error page. Might be more trouble than it's worth? https://stackoverflow.com/questions/6195805/jinja2-render-template-inheritance

stefanv commented:
Nice, thanks for the refactor!

kmshin1397 commented:
@acrellin this is ready for re-review/merge I think. I removed the old HTML file as you pointed out; also refactored to use Jinja template inheritance
